utils/logging_utils.py

```diff
@@ -28,8 +28,23 @@
 
 import importlib
 import logging
+import os
 
 combo_lock_logger = logging.getLogger("combo_lock")
 combo_lock_logger.disabled = True
 
-LOG = getattr(importlib.import_module("ovos_utils"), "LOG")
+
+def _init_app_logger():
+    logger = getattr(importlib.import_module("ovos_utils"), "LOG")
```

Comment:
Can this just be imported directly from `ovos.utils.log` instead of using importlib?

Reply:
> Can this just be imported directly from `ovos.utils.log` instead of using importlib?

Right, seems like a leftover, done